chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/CertifiedProductController.java

```diff
@@ -409,13 +491,23 @@ CertifiedProductSearchBasicDetails getCertifiedProductByChplProductNumberBasic(
         return results;
     }
 
-    @ApiOperation(value = "Get all of the Measures for a specified certified product based on a legacy "
+    @Operation(summary = "Get all of the Measures for a specified certified product based on a legacy "
             + "CHPL Product Number.",
-            notes = "\"Returns all of the Measures in the CHPL related to the specified certified product.  "
+            description = "\"Returns all of the Measures in the CHPL related to the specified certified product.  "
```

Comment:
```suggestion
            description = "Returns all of the Measures in the CHPL related to the specified certified product.  "
```

The other endpoints don't seem to have this escaped `"`